front/src/components/messaging/MessageBubbleEmployer.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Text, useTheme } from 'react-native-paper';
+
+import { Message } from '@/models/entities/message';
+
+/**
+ * The styles for the MessageBubbleEmployer component.
+ */
+const styles = StyleSheet.create({
+  container: {
+    alignSelf: 'flex-start',
+  },
+  message: {
+    borderRadius: 10,
+    marginBottom: 30,
```

Comment:
Spacing between bubbles should be controller by the `MessageList` component that displays them.